recipes/restinio/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RestinioConan(ConanFile):
+    name = "restinio"
+    license = "BSD-3-CLAUSE"
+    homepage = "https://github.com/Stiffstream/restinio"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "RESTinio is a header-only C++14 library that gives you an embedded HTTP/Websocket server."
+    topics = ("http-server", "websockets", "rest", "tls-support")
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"use_boost": [True, False], "use_openssl": [True, False]}
+    default_options = {"use_boost": False, "use_openssl": False}
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
+
+    def requirements(self):
+        self.requires("http_parser/2.9.4")
+        self.requires("fmt/6.2.1")
+        self.requires("expected-lite/0.4.0")
+        self.requires("optional-lite/3.2.0")
+        self.requires("string-view-lite/1.3.0")
+        self.requires("variant-lite/1.2.2")
+
+        if self.options.use_boost:
+            self.requires("boost/1.73.0")
+        else:
+            self.requires("asio/1.14.1")
```

Comment:
Should wait for asio/1.16.1 (https://github.com/conan-io/conan-center-index/pull/1521).

Reply:
There's a 1.16.0 as well

Reply:
It's not listed in the supported versions : https://stiffstream.com/en/docs/restinio/0.6/obtaining.html

Reply:
It was merged today